### ISSUE TITLE
id_rsa required

### ISSUE THREAD

**Issue by AstraLuma** [Gruntfuggly/gerrit-view#15]
> gerrit-view: Error: config.privateKey does not exist at /Users/jbliss/.ssh/id_rsa

I only have an `id_ed25519` key and do not have an `id_rsa` key.

Talking to gerrit over SSH outside of gerrit-view works just fine..

**Comment by kishankarun** [comment]
try creating ssh key using ssh-keygen

**Comment by AstraLuma** [comment]
yeah, i don't want to create a lower security RSA key when I already have an ed25519 key.
